packages/iota/src/models/outputs/IAliasOutput.ts

```diff
@@ -21,7 +22,7 @@ export interface IAliasOutput extends ITypeBase<4>, ICommonOutput {
     /**
      * Unique identifier of the alias, which is the BLAKE2b-160 hash of the Output ID that created it.
      */
-    aliasId: string;
+    aliasId: HexEncodedString;
 
     /**
      * A counter that must increase by 1 every time the alias is state transitioned.
```

Comment:
Can't comment there, but `stateMetadata` should also be updated

Reply:
Fixed.